benchmarks/QS_low_scaling_postHF/README.md

```diff
@@ -0,0 +1,65 @@
+# Quickstep low-scaling RPA and SOS-MP2 gradients benchmark
+
+## Description
+
+This benchmark set showcases sparse tensor based low-scaling post-HF ENERGY_FORCE calculations. Both
+the RPA and SOS-MP2 methods are represented. The implementation relies on the RI approximation in the
+AO basis. For increased sparsity and performance, the overlap RI metric is used. All input files are
+of production level. The methods make use of the DBT and DBM libraries, which are GPU accelerated.
+
+## Files
+
+There are three folders for the simulation of periodic water systems of increasing size, from 32 to
+128 molecules. Both the RPA and the SOS-MP2 calculations restart from a PBE wavefunction, which has
+to be calculated first (H2O-#-PBE-TZ.inp). All calculations are based on the triple-zeta cc-TZ and
+RI_TZ basis sets. The SOS-MP2 correlation energy is calculated on top of a HF SCF, and the RPA
+correlation energy is calculated on top of a PBE SCF (without EXX).
+
+## Reference timings
+
+These benchmarks were run during the pilot phase of the LUMI-G supercomputer (December 2022), under
+git commit d8f36242127b9a828f127550ecd9613eefb3f1cc. All calculations were run with 8 GPUs, 16 MPI
```

Comment:
Thanks a lot for adding these benchmarks Augustin!

I'm a bit confused by the eight GPUs because AFAIK LUMI-G has only four GPUs per node.

Reply:
Mi250 is 2 [GCDs](https://docs.lumi-supercomputer.eu/hardware/compute/lumig/), so at the end Linux sees 8 GPUs in total

Reply:
Ah, ok. Thanks for clarifying!